src/main/java/teammates/common/datatransfer/attributes/AdminEmailAttributes.java

```diff
@@ -151,4 +167,43 @@ public String getFirstAddressReceiver() {
     public String getFirstGroupReceiver() {
         return getGroupReceiver().get(0);
     }
+
+    public static class Builder {
+        private final AdminEmailAttributes adminEmailAttributes;
+
+        public Builder(String subject, List<String> addressReceiver, List<String> groupReceiver,
+                       Text content, Date sendDate) {
+            adminEmailAttributes = new AdminEmailAttributes();
+            adminEmailAttributes.addressReceiver = addressReceiver;
```

Comment:
Call Assumption.assertNotNull(String message, Object... objects) method here to make sure that all required arguments are present.